x-pack/plugin/src/test/resources/rest-api-spec/test/index_lifecycle/10_basic.yml

```diff
@@ -6,6 +6,16 @@ setup:
 
 ---
 "Test Basic Policy CRUD":
+  - do:
+      catch: missing
+      xpack.index_lifecycle.get_lifecycle:
```

Comment:
added to walk the branch in TransportGetLifecycleAction that checks for null metadata